specs/electra/beacon-chain.md

```diff
@@ -1271,12 +1258,14 @@ def process_deposit_receipt(state: BeaconState, deposit_receipt: DepositReceipt)
     if state.deposit_receipts_start_index == UNSET_DEPOSIT_RECEIPTS_START_INDEX:
         state.deposit_receipts_start_index = deposit_receipt.index
 
```

Comment:
What about checking `is_valid_deposit_signature` here? Unless we specifically want to avoid burning a top-up with incorrect signature

Reply:
There are two things why do signature check as is:
1) Some infra might already have top-ups sent with zero signatures (worth checking tho)
2) Rate limiting a number of deposit signature checks per epoch is really nice, it alleviates the risk of delaying block processing if someone manages to pack a thousand deposits into it